pkg/services/pluginsintegration/clientmiddleware/logger_middleware.go

```diff
@@ -73,7 +73,19 @@ func (m *LoggerMiddleware) QueryData(ctx context.Context, req *backend.QueryData
 	var resp *backend.QueryDataResponse
 	err := m.logRequest(ctx, func(ctx context.Context) (innerErr error) {
 		resp, innerErr = m.next.QueryData(ctx, req)
-		return innerErr
+
+		if innerErr != nil {
+			return innerErr
+		}
+
+		ctxLogger := m.logger.FromContext(ctx)
+		for refID, dr := range resp.Responses {
+			if dr.Error != nil {
+				ctxLogger.Error("Partial data response error", "ref_id", refID, "error", dr.Error)
```

Comment:
Nit:

```suggestion
				ctxLogger.Error("Partial data response error", "refID", refID, "error", dr.Error)
```

I think camel case is preferred for log attributes.

https://github.com/grafana/grafana/blob/main/contribute/backend/instrumentation.md#naming-conventions

Not sure if `refID` vs `refId` tho

Reply:
Good point thanks. Confused with the tracing attribute naming conventions :)